src/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScanner.java

```diff
@@ -71,7 +71,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		    				"", // Other info
 		    				getSolution(), //Solution
 		    				getReference(), //References
-		    				xpbDirective,	// Evidence - Return the X-Powered-By Header info
+		    				"X-Powered-By: " + xpbDirective, // Evidence - Return the X-Powered-By Header info

```

Comment:
>  so attempting to re-assemble it manually could fail to match properly.

Completely agreed.

> extract the (whole) header (with a regular expression?)

This could be slightly tricky (given variations in the new line characters, header text, etc.), so unless absolutely necessary, why add the complexity? :)

I've updated the PR with a constant. Let me know if we really need a regex matching for complete text, I'll update my PR.